pbiviz.json

```diff
@@ -21,7 +21,7 @@
     "node_modules/lodash/lodash.min.js",
     "node_modules/d3/d3.min.js",
     "node_modules/globalize/lib/globalize.js",
-    "node_modules/globalize/lib/cultures/globalize.culture.en-US.js",
+    "node_modules/globalize/lib/cultures/globalize.cultures.js",
```

Comment:
Do we really need to include all of languages?
I think we would need to support [these langs](https://github.com/Microsoft/PowerBI-visuals-tools/blob/master/templates/visuals/.api/v1.10.0/schema.stringResources.json#L19)